apps/passport-server/migrations/38_known_tickets.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE known_public_keys (
+  public_key_name VARCHAR NOT NULL,
+  public_key_type VARCHAR NOT NULL,
+  public_key JSONB UNIQUE,
```

Comment:
I think this should also be a VARCHAR. Otherwise you're overfitting to the fact that this is usually an eddsa key.

Reply:
What would the VARCHAR contain?